libraries/GCS_MAVLink/GCS.h

```diff
@@ -355,6 +356,13 @@ class GCS_MAVLINK
     virtual int32_t global_position_int_alt() const;
     virtual int32_t global_position_int_relative_alt() const;
 
+    // these methods are called after vfr_hud_velned is updated
+    virtual bool vfr_hud_make_alt_relative() const { return false; }
+    virtual float vfr_hud_climbrate() const;
+    virtual float vfr_hud_airspeed() const;
+    virtual int16_t vfr_hud_throttle() const { return 0; }
```

Comment:
Do these really need to be named `vfr_hud`? I would call them `get_X`, close to what we have done to frame_type, base_mode, etc.

Reply:
`vfr_hud_make_alt_relative`, definitely not...
`vfr_hud_airspeed` - definitely not...
climbrate I'm not sure about and throttle it probably doesn't matter ATM.

So all-in-all, I think it's better to keep these as a group.  Opportunity to reuse is negligible - and when we do we can rename then.